src/worker.js

```diff
@@ -7,20 +7,17 @@ import BatchManager from './utils/BatchManager';
 
 let closing = false;
 
-export default (app, config, onServer, workerId) => {
-  let server;
-
-  // ===== Middleware =========================================================
+export const attachMiddleware = (app, config) => {
```

Comment:
now that this has named exports, babel's `.default` will now be present, which is no good.

Can we expose these things in the root entry point, to avoid people needing to deep link into `src/worker` or `lib/worker`, and thus avoid people needing to depend on `.default` (or avoid us needing to avoid `export` syntax here)?